watson/cli.py

```diff
@@ -371,6 +381,23 @@ def report(watson, from_, to, projects, tags):
             style('time', '{}'.format(format_timedelta(total)))
         ))
 
+    target_hours_per_week = watson.config.getfloat('options',
+                                                   'target_hours_per_week')
+    if target_hours_per_week is not None:
+        sec_per_min = 60
+        min_per_hr = 60
+        offset = 1000
+        weeks = abs(span.start - span.stop).days/7
```

Comment:
I just realized that this calculation isn't quite right. This works if there are 7 days in the workweek, but a typical week is 5 days.